ci/test/00_setup_env_s390x.sh

```diff
@@ -6,9 +6,10 @@
 
 export LC_ALL=C.UTF-8
 
-export HOST=s390x-unknown-linux-gnu
```

Comment:
Why?

I presume because the packages are not available?

https://packages.ubuntu.com/eoan/g++-multilib-s390x-linux-gnu
https://packages.ubuntu.com/eoan/gcc-s390x-linux-gnu

Reply:
If so, this should be reported as a bug. Looks like at least debian is working on them: https://packages.debian.org/experimental/gcc-s390x-linux-gnu

Reply:
Yes. that's the reason

Reply:
Hmm. If the host is not set, it defaults to the arch this script is executed on. This should be the same.

What is the error you were seeing when leaving this line as is?

Reply:
On travis this is set to `s390x-ibm-linux-gnu`

Reply:
that it can't find the `gcc-s390x-linux-gnu` package

Reply:
Eh? You mean `error: provided command 's390x-unknown-linux-gnu-g++' not found`?

If so, `export HOST=s390x-linux-gnu` should work

Reply:
Sorry, forget my comment. The cross compile packages are missing either way on s390x

Reply:
On a second try, I think installing gcc on s390x will create the appropriate symlinks in `/usr/bin/s390x-linux-gnu-gcc -> gcc-9`

This should work:

```diff
diff --git a/ci/test/00_setup_env_s390x.sh b/ci/test/00_setup_env_s390x.sh
index c17c890546..637d549553 100644
--- a/ci/test/00_setup_env_s390x.sh
+++ b/ci/test/00_setup_env_s390x.sh
@@ -6,10 +6,11 @@
 
 export LC_ALL=C.UTF-8
 
+export HOST=s390x-linux-gnu
 # The host arch is unknown, so we run the tests through qemu.
 # If the host is s390x and wants to run the tests natively, it can set QEMU_USER_CMD to the empty string.
 export QEMU_USER_CMD="${QEMU_USER_CMD:"qemu-s390x"}"
-export PACKAGES="python3 python3-zmq bsdmainutils qemu-user"
+export PACKAGES="python3-zmq bsdmainutils qemu-user"
 export RUN_UNIT_TESTS=true
 export RUN_FUNCTIONAL_TESTS=true
 export GOAL="install"